test/unit/commands/processargs-specs.js

```diff
@@ -0,0 +1,66 @@
+import sinon from 'sinon';
+import XCUITestDriver from '../../..';
+
+
+describe('process args', () => {
+  let driver = new XCUITestDriver();
+  let proxySpy = sinon.spy(driver, 'proxyCommand');
+
+  const PROCESS_ARGS_OBJECT = {
+    args: ["a", "b", "c"]
+    , env: { "a": "b", "c": "d" }
+  };
+  const PROCESS_ARGS_STRING = "{\"args\": [\"a\", \"b\", \"c\"], \"env\": { \"a\": \"b\", \"c\": \"d\" }}";
```

Comment:
This can be be more easily (and maintainably) as `JSON.stringify(PROCESS_ARGS_OBJECT);`